frontend/plugins/tourism_ui/src/modules/tms/branch-detail/dashboard/tours/_components/TourCreateForm.tsx

```
@@ -115,9 +127,69 @@ export const TourCreateForm = ({ branchId, onSuccess }: Props) => {
+    const current = form.getValues('translations') || [];
+    const currentLangs = current.map((t) => t.language);
+    const pricingOptionIds = (form.getValues('pricingOptions') || []).map(
+      (p) => p._id,
+    );
+    if (!translationLanguages.every((l) => currentLangs.includes(l))) {
+      form.setValue(
+        'translations',
+        buildEmptyTourTranslations(translationLanguages, pricingOptionIds),
+      );
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)